### PR TITLE
Completes exercises

### PR DESCRIPTION
**Pull request by janmilosh**

### CODE REVIEW

File: lib/fizzbuzz.rb [hunk 1]
```diff
@@ -1,11 +1,21 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+    if input % 15 == 0
```
Comment (degoeym) [inline lib/fizzbuzz.rb]: Nice. :+1:

File: spec/deaf_grandma_spec.rb [hunk 2]
```diff
@@ -9,10 +10,12 @@
...
-    #implement your test here
+    expect(script.speak('HI GRANDMA')).to eq 'NOT SINCE 1964!'
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    script.speak('BYE')
+    script.speak('BYE')
+    expect(script.speak('BYE')).to eq 'SEE YOU LATER SONNY!'
   end
```
Comment (degoeym) [inline spec/deaf_grandma_spec.rb]: This test can be tightened up a bit.

File: lib/deaf_grandma.rb [hunk 3]
```diff
@@ -19,9 +19,17 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    if input.match(/BYE/) and @bye_counter == 2
+      @bye_counter = 0
+      'SEE YOU LATER SONNY!'
+    elsif input.match(/^[^a-z]*$/)
```
Comment (degoeym) [inline lib/deaf_grandma.rb]: :tada:

**Comment by nicolasmccurdy** [comment]
:+1:

File: spec/deaf_grandma_spec.rb [hunk 4]
```diff
@@ -9,10 +10,12 @@
...
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak('HI GRANDMA')).to eq 'NOT SINCE 1964!'
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    expect(script.speak('BYE')).to eq 'NOT SINCE 1964!'
+    expect(script.speak('BYE')).to eq 'NOT SINCE 1964!'
```
Comment (degoeym) [inline spec/deaf_grandma_spec.rb]: Better, but it can still be tightened up a bit more.

**Pull request closed without merging**